pre_install_report/library/pre_install_utils.py

```diff
@@ -77,12 +77,12 @@ def do_cmd(self, test_cmd):
 
             self.logger.info("cmd {} rc = 0".format(test_cmd))
             self.logger.debug("cmd {} rc = 0 response {}".format(test_cmd, str(data)))
-            return 0
+            return 0, str(data)
```

Comment:
Are consumers/users of do_cmd() processing both rc and data returned?

Reply:
Yes.  If needed I can incorporate the message returned into the Report.

Reply:
Does that mean there is no need, currently, to incorporate the returned message into the report?